README.md

```diff
@@ -1,2 +1,38 @@
-# seneca-entity
-Entity plugin for seneca
+![Seneca](http://senecajs.org/files/assets/seneca-logo.png)
+> A Node.js toolkit for Micro-Service Architectures
+
+# seneca
+[![npm version][npm-badge]][npm-url]
+[![Build Status][travis-badge]][travis-url]
+
+## About Seneca Entity
+_Seneca Entity_ is an entity plugin for Seneca
+
+## Install
+To install, simply use npm.
+
+```
+npm install seneca-entity
+```
+
+## Test
+To run tests, simply use npm:
+
+```
+npm run test
+```
+## Contributors
```

Comment:
Yeah, don't use this, it should look closer to https://github.com/senecajs/seneca-level-store without the annotated or data specifc sections.